lib/ruby_drills/array/select_drill.rb

```diff
@@ -0,0 +1,31 @@
+class SelectDrill < Drill
+
+  def setup
+    @values = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34]
+    @hints = ["Are you being selective enough?",
+              "This method is called filter in other languages.",
+              "http://ruby-doc.org//core-2.2.0/Enumerable.html#method-i-select"]
```

Comment:
This should be "http://ruby-doc.org/core/Enumerable.html#method-i-select" instead.